.vscode/settings.json

```diff
@@ -5,8 +5,8 @@
     "**/.hg": true,
     "**/CVS": true,
     "**/.DS_Store": true,
-    "node_modules": true,
-    ".next": true
+    "node_modules": false,
+    ".next": false
```

Comment:
Sometimes, we want to delete cache and node_modules manually, so I changed this place 🙏